tests/test_modeling_big_bird.py

```diff
@@ -398,6 +398,11 @@ class BigBirdModelTest(ModelTesterMixin, unittest.TestCase):
     # head masking & pruning is currently not supported for big bird
     test_head_masking = False
     test_pruning = False
+
+    # torchscript should be possible, but takes prohibitively long to test.
+    # Also torchscript is not an important feature to have in the beginning.
+    test_torchscript = False
```

Comment:
The torchscript tests are really slow. Running those at every commit is too costly at the moment -> let's comment them out. Also the model will probably not be used with torchscript in the beginning. We can revisit this at a later point.